src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3429,4 +3429,7 @@
   <data name="ShortcutItemWorkingDir" xml:space="preserve">
     <value>Start in:</value>
   </data>
+  <data name="FileTooLargeHeader" xml:space="preserve">
+    <value>The following files are too big to be copied on ({0}):</value>
```

Comment:
```suggestion
    <value>The following files are too large to be copied to ({0}):</value>
```
Would it be worth adding a comment explaining what {0} is?

Reply:
`{0} is the root path of the target drive` something like this?

Reply:
Yes